Tactics/Simp.lean

```diff
@@ -113,4 +116,21 @@ def LNSymSimp (goal : MVarId)
     | none => return none
     | some (_, goal') => return goal'
 
+/--
+Invoke `simp [..] at *` at the given goal `g` with
+simp context `ctx` and simprocs `simprocs`.
+-/
+def LNSymSimpAtStar (g : MVarId)
+    (ctx : Simp.Context)
+    (simprocs : Array Simp.Simprocs)
+    : MetaM (Option MVarId) := do
+   g.withContext do
+    let fvars : Array FVarId :=
+      (← getLCtx).foldl (init := #[]) fun fvars d => fvars.push d.fvarId
+    let (result, _stats) ← simpGoal g ctx simprocs (fvarIdsToSimp := fvars)
+      (simplifyTarget := true) (discharge? := none)
+    match result with
+    | none => return none
+    | some (_newHyps, g') => pure g'
+
```

Comment:
Should we add an argument similar to [`Tactic.Location`](https://leanprover-community.github.io/mathlib4_docs/Lean/Elab/Tactic/Location.html#Lean.Elab.Tactic.Location) to LNSymSimp instead of creating a new single-purpose def?

Location probably doesn't work as-is, because it takes explicitly provided local variables as `Syntax` for some reason, but we can copy the shape of the type

Reply:
@alexkeizer I'm not sure, to be honest, I prefer this separation, because it makes it clear that `LNSymSimpAtStar` is a think wrapper around the existing `LNSymSimp`.